credentials/credentials.go

```diff
@@ -287,5 +287,5 @@ type ChannelzSecurityValue interface {
 type OtherChannelzSecurityValue struct {
 	ChannelzSecurityValue
 	Name  string
-	Value protoadapt.MessageV1
```

Comment:
This may be a breaking change.  It is probably fine since it's experimental, and it's unlikely anyone is using it....so I think I'm OK with it.  Just wanted to call it out.